### PR TITLE
Feature/incident feedback digest

### PR DESCRIPTION
**Pull request by mvilanova**
This PR adds a daily digest for incident feedback. The digest is emailed to the commander.

Screen Shot 2020-11-19 at 3 36 28 PM

### CODE REVIEW

File: src/dispatch/feedback/service.py [hunk 1]
```diff
@@ -14,6 +15,19 @@ def get_all(*, db_session):
...
 
+def get_all_last_x_hours(
+    *, db_session, hours: int = 24, skip=0, limit=100
+) -> List[Optional[Feedback]]:
+    """Returns all feedback provided in the last x hours."""
+    return (
+        db_session.query(Feedback)
+        .filter(Feedback.created_at >= datetime.utcnow() - timedelta(hours=hours))
+        .offset(skip)
+        .limit(limit)
```
Comment (kevgliss) [inline src/dispatch/feedback/service.py:26]: Why do we limit this? You don't seem to use pagination anywhere. Same with skip/offset.

File: src/dispatch/feedback/messaging.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                "rating": piece.rating,
+                "feedback": piece.feedback,
+                "participant": participant,
+                "created_at": piece.created_at,
+            }
+        )
+
+    name = subject = "Incident Feedback Daily Digest"
+    plugin.instance.send(
+        commander_email,
```
Comment (kevgliss) [inline src/dispatch/feedback/messaging.py:49]: Did we want to add support to cc the security team?

**Review by kevgliss — APPROVED** [review]

File: src/dispatch/feedback/messaging.py (hunk 2 above)
Comment (mvilanova) [reply src/dispatch/feedback/messaging.py:49]: Yes, I was going to work on it in a separate PR.

File: src/dispatch/feedback/service.py (hunk 1 above)
Comment (mvilanova) [reply src/dispatch/feedback/service.py:26]: Correct, not needed, will remove.

**Pull request merged**